services/prometheus/internal/common/config/http_config.go

```diff
@@ -14,7 +14,7 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
-// Original file: https://github.com/prometheus/common/blob/v0.9.1/config/http_config.go
+// Original file: https://github.com/prometheus/common/blob/v0.8.0/config/http_config.go
```

Comment:
This file in both versions is identical, but Prometheus v1.16.0 depends on common v0.8.0.
So comment was updated to match consistent dependencies state.